floodviz/static/js/hydrograph.js

```diff
@@ -50,7 +58,7 @@ document.addEventListener("DOMContentLoaded", function (event) {
 				.attr("transform",
 					"translate(" + margin.left + "," + margin.top + ")");
 
-			data.forEach(function (d) {
+			data.map(function(d) {
```

Comment:
So the map function should return the mapped value. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map/
So this should look something like this
```
graphData = data.map(function(d) { 
     return Number(d.value);
}
```
This way you don't change data the parameter passed in. You create a new array for the graph.